CAM_interface/nn_interface_CAM.F90

```diff
@@ -790,13 +814,10 @@ subroutine SAM_var_conversion(t, r, tabs, qv, qc, qi)
             ! Set tabs to iterated tabs after convection
             tabs(i,k) = tabs1
 
-            ! Code for calculating qc and qi from rn.
-            ! Adapted from statistics.f90 in SAM assuming dokruegermicro=.false.
-            ! Also implements conversion from SAM dry mixing ratios to CAM moist mixing ratios
+            ! Calculating qc and qi from rn and converst from SAM dry
```

Comment:
Typo - 'convert'

Also arguably misuse of 'and' -- calculating qc and qi from rn _is_ converting from dry to moist, not a separate operation.

Reply:
corrected in https://github.com/m2lines/convection-parameterization-in-CAM/pull/80/commits/fa143448df385b90f60d06939b7ea73acd90e443